ppr-ui/src/utils/auth-helper.ts

```diff
@@ -41,65 +41,68 @@ export function getKeycloakRoles (): Array<string> {
   throw new Error('Error getting Keycloak roles')
 }
 
+export async function getStaffegisteringParty (isBcOnline: boolean): Promise<PartyIF> {
+  let partyCode = sessionStorage.getItem(SessionStorageKeys.PprStaffPartyCode)
```

Comment:
SessionStorageKeys is just for the common components (it doesn't include our own config stuff that we put in session storage). I don't think we have an enum or anything for it so you just need to do the string *PPR_STAFF_PARTY_CODE*

Same for below

Reply:
gotcha, made the change